turbo/trie/witness.go

```diff
@@ -15,7 +15,7 @@ type WitnessStorage interface {
 // WitnessVersion represents the current version of the block witness
 // in case of incompatible changes it should be updated and the code to migrate the
 // old witness format should be present
-const WitnessVersion = uint8(1)
+const WitnessVersion = uint8(0)
```

Comment:
This is the mpt specific change right?

Reply:
Yes correct. This repo only supports MPT. For SMT, it will be in https://github.com/0xPolygonHermez/cdk-erigon.

Reply:
Yeah, I just remembered the convo vaguely from yesterday and wanted to be sure. I guess we should document this somewhere. Probably in the zk_evm stack though since we'll need to handle it on that side.